### PR TITLE
Event backend update

### PR DESCRIPTION
**Pull request by nickchrysaf**
### Add
-Added Slots field in Event model.
-Added Date filtering method.

### Issuer
#73

### CODE REVIEW

File: backend/eatandmeet/api/models.py [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 from authentication.models import User
 
 
 TOPIC_CHOICES = [
         ('Politics', 'Politics'),
         ('Soccer', 'Soccer'),
-        ('Films' ,'Films'),
-        ('Music' ,'Music'),
+        ('Films','Films'),
+        ('Music','Music'),
```
Comment (lysnikolaou) [inline backend/eatandmeet/api/models.py]: Add the rest of the topics, please!

File: backend/eatandmeet/api/models.py [hunk 2]
```diff
@@ -25,11 +26,20 @@ class Event(models.Model):
     event_members = models.ManyToManyField(User, related_name='event_members', blank=True)
     topics = MultiSelectField(choices=TOPIC_CHOICES)
     location = models.CharField(max_length=11, choices =PLACE_CHOICES, default ='Hauptmensa' )
```
Comment (lysnikolaou) [inline backend/eatandmeet/api/models.py]: Also remove the voices here. Let it be just a string.

```suggestion
    location = models.CharField(max_length=30, default ='Hauptmensa' )
```

File: backend/eatandmeet/api/models.py [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
+        ('Science','Science'),
+        ('Animals','Animals'),
+        ('Celebrities','Celebrities'),
+        ('News','News'),
+        ('General','General'),
+        ('Smalltalk','Smalltalk'),
+        
     ]
 
 PLACE_CHOICES = [
```
Comment (lysnikolaou) [inline backend/eatandmeet/api/models.py]: Not that important, but this list can be removed now.

File: backend/eatandmeet/api/models.py [hunk 4]
```diff
@@ -1,14 +1,27 @@
...
 from authentication.models import User
 
 
 TOPIC_CHOICES = [
-        ('Politics', 'Politics'),
-        ('Soccer', 'Soccer'),
-        ('Films' ,'Films'),
-        ('Music' ,'Music'),
+        
+        ('Politics','Politics'),
```
Comment (nitzanashi) [inline backend/eatandmeet/api/models.py]:
```suggestion
        ('Politics','Politics'),
        ('Environment','Environment')
```

**Pull request merged**